docs/configuring.md

```diff
@@ -327,14 +327,14 @@ The Theme Manager expects to be able to load theme CSS/font assets from
 Continuing the example above:
 
 ```bash
-cd $YOUR_JUPYTERLITE/lab/extensions
-mkdir -p ../build/themes
-cp -r @*/*/themes/* ../build/themes/
-cp -r @*/themes/* ../build/themes/
+cd $YOUR_JUPYTERLITE/extensions
+mkdir -p ../../build/themes
+cp -r @*/*/themes/* ../../build/themes/
+cp -r @*/themes/* ../../build/themes/
 # To also ensure these are available for JupyterLite Retro:
-mkdir -p ../../retro/build/themes
-cp -r @*/*/themes/* ../../retro/build/themes/
-cp -r @*/themes/* ../../retro/build/themes/
+mkdir -p ../../build/themes
```

Comment:
is the second stanza still needed?

Reply:
Should be fixed in https://github.com/jupyterlite/jupyterlite/pull/472/commits/8b6694239c3d254a6a7891a21ca97628b9ad7f98